src/Process/Factory/MutantProcessContainerFactory.php

```diff
@@ -61,6 +67,8 @@ public function __construct(
 
     public function create(Mutant $mutant, string $testFrameworkExtraOptions = ''): MutantProcessContainer
     {
+        $timeout = max(self::MIN_TIMEOUT, min(self::TIMEOUT_FACTOR * $mutant->getMutation()->getNominalTestExecutionTime(), $this->timeout));
```

Comment:
just for all of us to make it clear: this function currently returns the sum of all covering tests, not individual test time

we have another issue for `S`kipped mutants here: https://github.com/infection/infection/issues/1965

This doesn't block this PR, but wanted to highlight here that we still have some confusing logic for timeouts for `S`kipped mutants